### ISSUE TITLE
[202203013]  Boostcamp-AI-Casual-Talk - 0회차

### ISSUE THREAD

**Issue by ili0820** [ili0820/Boostcamp-AI-Casual-Talk#1]
- News
  - [신한은행, '인공지능 ATM'으로 보이스피싱 막는다](https://www.yna.co.kr/view/AKR20220307075300002?input=1195m)
  - [시연](https://www.yna.co.kr/view/MYH20220312001300641)
     ATM에 장착된 카메라로 이상행동 감지!
  - [8 Ways to Prevent Ageism in Artificial Intelligence](https://www.inc.com/anna-meyer/ageism-artificial-intelligence-world-health-organization.html)
         a. Include older consumers in the design of A.I. technologies.
         b. Hire age-diverse individuals for data science teams.
         c. Conduct age-inclusive data collection.
         d. Invest in digital infrastructure and digital literacy.
         e. Give older consumers the right to consent and contest.
         f. Work alongside governance frameworks and regulations.
         g. Stay up to date on the new uses of A.I. and how to avoid bias.
         h. Create robust ethics processes.
    - WHO report (2022.2): https://www.who.int/publications/i/item/9789240040793
    - 예시)[클로바 케어콜](https://www.navercorp.com/promotion/pressReleasesView/30727)
  - [RecSys Cahllenge 2022 시작!](http://www.recsyschallenge.com/2022/)
 

![image](https://user-images.githubusercontent.com/65278309/158043565-42fc496f-5b0a-45a1-8e1a-622d32dce962.png)

- ArXiv
  - [L-Verse: Bidirectional Generation Between Image and Text](http://arxiv.org/abs/2111.11133v7)
 ![image](https://user-images.githubusercontent.com/65278309/158060968-407d8654-9225-420a-9d1d-0e59ee8a791c.png)
![image](https://user-images.githubusercontent.com/65278309/157776822-9b4fb67f-4280-4795-84e1-44767b0cdd7c.png)
image to text, text to image 둘 다 학습!
  - [Quality-aware News Recommendation](https://arxiv.org/abs/2202.13605v1)
  
![image](https://user-images.githubusercontent.com/65278309/158061339-82d77325-78f1-4979-b8ab-381d4335d4b0.png)

**Comment by hawe66** [comment]
* AI-inspired cognitive science?  
  * [Richard Sutton's recent Twitter posting](https://twitter.com/RichardSSutton/status/1499200262937845760)  
  * **Reinforcement Learning**  
  * Kalman Filter (Control Theory)  
  * Utility Theory (Economics)  
  * [The Multi-disciplinary Conference on Reinforcement Learning and Decision Making](https://rldm.org/)  
    * Researchers from computer science, neuroscience, economics and others gather and share their domain knowledge!!!
* Arxiv  
[The Quest for a Common Model of the Intelligent Decision Maker](https://t.co/F1HgNDp5xo)
![Intelligence_Components](https://user-images.githubusercontent.com/67923359/157788412-6697778d-176d-4ea3-8850-d535cb862e66.JPG)

[Slides](https://docs.google.com/presentation/d/1sDkZQduDp1P-Pow8jAPs3Gq6TyzXkZLg_E9EhqA3Knk/edit#slide=id.p)

**Comment by ijkimmy** [comment]
- arxiv
    - [[Model soups: averaging weights of multiple fine-tuned models improves accuracy without increasing inference time](https://arxiv.org/pdf/2203.05482v1.pdf)](https://arxiv.org/pdf/2203.05482v1.pdf)
        - averaging the weights of multiple models fine-tined with different hyperparameter configurations often improves accuracy and robustness (so-called *model soup*)
        - new state of the art for ImageNet-1k
        - Inspired by [[What is being transferred in transfer learning?](https://arxiv.org/pdf/2008.11687.pdf)](https://arxiv.org/pdf/2008.11687.pdf)
            - **Models trained from pre-trained weights** make similar mistakes on target domain, have similar features and are surprisingly close in l2-distance in parameter space (**same basin of the loss landscape**), whereas models from random initialization don’t.
        - Weight averaging along a single training trajectory has previously shown to improve the performance of the model trained from random initialization
        - model soup composed of ViT-G became new SOTA, while requiring 25% fewer FLOPs at inference time
        - Amongst uniform, greedy, learned model soup, the greedy soup is the central method.  
        - [참고](https://jtuckerk.github.io/loss_landscape.html)
  
![Untitled](https://user-images.githubusercontent.com/30295190/158015462-9c297459-2f15-4db6-b715-4eccf5b36d57.png)

**Comment by sunbi-s** [comment]
Metaverse and  NFT (Non-Fungible Token)
- [엑시인피니티, NFT 게임 시장 70% 차지](https://www.blockmedia.co.kr/archives/217389)
- ['NFT 대장주' 엑시인피니티란 무엇](http://news.bizwatch.co.kr/article/industry/2022/02/21/0018)
- [MS의 82조원짜리 M&A가 보여준 메타버스의 미래](https://www.chosun.com/economy/int_economy/2022/01/20/CVLYPQDUA5BHJPEER4XZSBAE5E/)
![image](https://user-images.githubusercontent.com/78129508/158047493-f47a41d0-88b3-4d64-a9ba-eb681ab71206.png)
![image](https://user-images.githubusercontent.com/78129508/158047498-1f7802e9-f0dd-400a-958a-63eb345dc865.png)

**Comment by ili0820** [comment]
다들 고생하셨습니다!! 좋은 밤되세요!
